web-tech-exercises/5/movies.js

```diff
@@ -0,0 +1,129 @@
+const apiKey = '648e752c';
+let fetchedMovies = [];
+let globalSortOrder='asc';
+let page=1;
+let search='';
+ 
+const fetchMovies = async (searchTerm, page=1) => {
+    try {
+        const response = await fetch(`https://www.omdbapi.com/?s=${searchTerm}&apikey=${apiKey}&page=${page}`);
+        const data = await response.json();
+ 
+        if (data.Response === "True") {
+            if(page===1){
+                fetchedMovies=data.Search;
+                document.getElementById('load-more').style.display='block';
+                document.getElementById('secondForm').style.display='block';
+            }
+            else{
+                fetchedMovies=[...fetchedMovies, ...data.Search];
+            }
+            sortedMovies=sortMoviesByYear(fetchedMovies,globalSortOrder);
+            showResults(sortedMovies);
+            return fetchedMovies;
+        } else {
+            console.error("No movies found!");
+            document.getElementById('load-more').style.display='none';
+            return [];
+        }
+    } catch (error) {
+        console.error("Error fetching movies:", error);
+        return [];
+    }
+};
+ 
+const sortMoviesByYear = (movies, order) => {
+    if (order === 'asc') {
+        return movies.sort((a, b) => parseInt(a.Year) - parseInt(b.Year));
+    }
+    else if (order === 'desc') {
```

Comment:
last block `if-else` ladder should be `else`

Reply:
done ✅